src/lib/utils/esdm/info.txt

```diff
@@ -0,0 +1,13 @@
+<defines>
+ESDM -> 20240814
+</defines>
+
+<module_info>
+name -> "ESDM"
+</module_info>
+
+load_on vendor
+
+<libs>
+linux -> esdm_rpc_client
+</libs>
```

Comment:
Do we need this? Couldn't we just `load_on vendor` the actual `esdm_rng` module?
If that's just a naming issues (for `--with-esdm`), I guess we could just rename the module in "rng" accordingly.

Reply:
Changed this to with-esdm_rng and dropped the esdm info file.